### ISSUE TITLE
CC a11y: publish breadcrumb checklist - create PR

### ISSUE THREAD

**Issue by cathybaptista** [uswds/uswds-site#2854]
Publish the usa-breadcrumb accessibility test checklist items found in the [Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing) (Google Doc 🔒).

The process for creating a new accessibility test page can be found in the [Creating accessibility checklists - process doc](https://docs.google.com/document/d/14WTCsdzgwpoz_Miztd66RmEGfwwSEUFCVN1DkSQITTE/edit#heading=h.fa3sv1ray42g) (Google Doc 🔒).

**Comment by amyleadem** [comment]
@amycole501 This appears to have been completed with #2853. Moving this issue to fed final review because it looks like it should be good to close.
